releasenotes/notes/add-admission-controller-f88d88bd56d870f4.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Create server endpoint in cluster agent kubernetes admission controller
```

We should be more clear and explanatory here. This is a message for our users so that they know what they are getting. Would be nice to explain it.

```suggestion
    Create server endpoint in cluster agent kubernetes admission controller.
```